sormas-backend/src/main/java/de/symeda/sormas/backend/user/UserRoleFacadeEjb.java

```diff
@@ -254,6 +297,243 @@ public Map<UserRoleDto, Set<UserRight>> getUserRoleRights() {
+		descHeadlineCell.setCellStyle(boldStyle);
+
+		sheet.setColumnWidth(0, 256 * 35);
+		sheet.setColumnWidth(1, 256 * 50);
+		sheet.setColumnWidth(2, 256 * 50);
+		sheet.createFreezePane(2, 2, 2, 2);
+
+		int columnIndex = 3;
+
+		for (UserRight userRight : UserRight.values()) {
+			String columnCaption = userRight.name();
+			Cell headerCell = headerRow.createCell(columnIndex);
+			headerCell.setCellValue(columnCaption);
+			headerCell.setCellStyle(boldStyle);
+			sheet.setColumnWidth(columnIndex, 256 * 14);
+			columnIndex++;
+		}
+
+		Cell uuidHeadlineCell = headerRow.createCell(columnIndex++);
+		uuidHeadlineCell.setCellValue(I18nProperties.getCaption(Captions.UserRole_uuid));
+		uuidHeadlineCell.setCellStyle(boldStyle);
+
+		Cell portHealthUserHeadlineCell = headerRow.createCell(columnIndex++);
+		portHealthUserHeadlineCell.setCellValue(I18nProperties.getCaption(Captions.UserRole_portHealthUser));
+		portHealthUserHeadlineCell.setCellStyle(boldStyle);
+
+		Cell hasAssociatedDistrictUserHeadlineCell = headerRow.createCell(columnIndex++);
+		hasAssociatedDistrictUserHeadlineCell.setCellValue(I18nProperties.getCaption(Captions.UserRole_hasAssociatedDistrictUser));
+		hasAssociatedDistrictUserHeadlineCell.setCellStyle(boldStyle);
+
+		Cell hasOptionalHealthFacilityHeadlineCell = headerRow.createCell(columnIndex++);
+		hasOptionalHealthFacilityHeadlineCell.setCellValue(I18nProperties.getCaption(Captions.UserRole_hasOptionalHealthFacility));
+		hasOptionalHealthFacilityHeadlineCell.setCellStyle(boldStyle);
+
+		Cell enabledHeadlineCell = headerRow.createCell(columnIndex++);
+		enabledHeadlineCell.setCellValue(I18nProperties.getCaption(Captions.UserRole_enabled));
+		enabledHeadlineCell.setCellStyle(boldStyle);
+
+		//User roles rows
+		for (UserRoleDto userRole : userRoleRights.keySet()) {
+			Row row = sheet.createRow(rowCounter++);
+
+			Cell nameCell = row.createCell(0);
+			nameCell.setCellValue(userRole.getCaption());
+			nameCell.setCellStyle(boldStyle);
+
+			Cell captionCell = row.createCell(1);
+			captionCell.setCellValue(userRole.getJurisdictionLevel().toString());
+
+			Cell descCell = row.createCell(2);
+			descCell.setCellValue(userRole.getDescription());
+
+			columnIndex = 3;
+			for (UserRight userRight : UserRight.values()) {
+				Cell roleRightCell = row.createCell(columnIndex);
+
+				if (hasUserRight(Collections.singletonList(userRole), userRight)) {
+					roleRightCell.setCellStyle(authorizedStyle);
+					roleRightCell.setCellValue(I18nProperties.getString(Strings.yes));
+				} else {
+					roleRightCell.setCellStyle(unauthorizedStyle);
+					roleRightCell.setCellValue(I18nProperties.getString(Strings.no));
+				}
+				columnIndex++;
+			}
+
+			Cell uuidCell = row.createCell(columnIndex++);
+			uuidCell.setCellValue(userRole.getUuid());
+
+			Cell portHealthUserCell = row.createCell(columnIndex++);
+			portHealthUserCell.setCellValue(userRole.isPortHealthUser());
```

Comment:
boolean values should be translated to yes/no similarly to user rights

do it for this and the next few cells